packages/camera/camera/README.md

```diff
@@ -88,10 +88,16 @@ Here is a list of all permission error codes that can be thrown:
 
 - `CameraAccessDenied`: Thrown when user denies the camera access permission.
 
-- `CameraAccessDeniedWithoutPrompt`: iOS only for now. Thrown when user has previously denied the permission. iOS does not allow prompting alert dialog a second time. Users will have to go to Settings > Privacy in order to enable camera access.
+- `CameraAccessDeniedWithoutPrompt`: iOS only for now. Thrown when user has previously denied the permission. iOS does not allow prompting alert dialog a second time. Users will have to go to Settings > Privacy > Camera in order to enable camera access.
 
 - `CameraAccessRestricted`: iOS only for now. Thrown when camera access is restricted and users cannot grant permission (parental control).
 
+- `AudioAccessDenied`: Thrown when user denies the audio access permission.
```

Comment:
So per your design doc edits this will also be implemented for Android?
https://github.com/flutter/plugins/blob/7686be7ecbe1cc049423715b5e2d897cb63a43c1/packages/camera/camera/android/src/main/java/io/flutter/plugins/camera/CameraPermissions.java#L29

Reply:
Yep, android also handles camera and audio permissions separately.